alkira/resource_alkira_credential_pan_instance.go

```diff
@@ -16,6 +16,7 @@ func resourceAlkiraCredentialPanInstance() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			State: schema.ImportStatePassthrough,
 		},
+		DeprecationMessage: "Pan instance credentials to be moved into the alkira_service_pan resource. Only need Name, Auth_Code and Auth_Key.",
```

Comment:
Could we rephrase the message to be a bit clearer? Like this

" credential_pan_instance has been deprecated. Please specify `auth_code` and `auth_key` directly in `instance` block of  resource `service_pan`.